firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/common/APIController.kt

```diff
@@ -106,31 +108,33 @@ internal constructor(
       install(ContentNegotiation) { json(JSON) }
     }
 
-  suspend fun generateContent(request: GenerateContentRequest): GenerateContentResponse =
+  suspend fun generateContent(request: GenerateContentRequest): GenerateContentResponse.Internal =
```

Comment:
This API is not super consistent here, since it receives a non-internal type but returns an internal one. Was that the way it always work?  Should we reconsider?

This applies to several methods within APIController

Reply:
The request types are internal in common, the changes from `Foo` to `Foo.Internal` matches our renaming. Originally, the common types were called the same thing as the API types, now they're inner classes in the API types, this is basically the same method as it was. We could rename types if we wanted, thoughts?